hack/test-cmd.sh

```diff
@@ -329,10 +329,11 @@ oc secrets new-dockercfg dockercfg --docker-username=sample-user --docker-passwo
 oc describe secrets/dockercfg | grep "dockercfg:" | awk '{print $2}' > ${HOME}/.dockercfg
 oc secrets new from-file ${HOME}/.dockercfg
```

Comment:
change this to a different filename and use your new key mapping syntax to make the auto-detection work correctly.